_gitbook/syntax_and_semantics/macros/macro_methods.md

```diff
@@ -20,3 +20,19 @@ person.instance_vars_names #=> ["name", "age"]
 
 Note that in the case of macro defs you need to specify the return type.
 
+Arguments in macro defs are evaluated in running time, not a compile time:
```

Comment:
It's true for all macro's though, mentioning it only for macro defs could only replace it with another misconception :)